tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: prepare-repo
+      image: quay.io/konflux-ci/release-service-utils:4576e1e2a30439129cb69c8a4f39f7ced2d44376
+      script: |
+        set -eux
+
+        MRRC_FILE="$(workspaces.data.path)/$(params.mrrcParamFilePath)"
+        . "$MRRC_FILE"
+        echo "Downloading the maven repo zip $MRRC_ZIP_REGISTRY"
+        mrrc_dir="$(dirname $MRRC_FILE)/mrrc"
+        mkdir -p $mrrc_dir
+        cd $mrrc_dir
+        oras pull $MRRC_ZIP_REGISTRY
```

Comment:
oras has been giving us lots of issues because it isn't great at figuring out auth. I think you will need to do something like this https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/create-pyxis-image/create-pyxis-image.yaml#L118

Reply:
Ok. Will check.

Reply:
Changed accordingly in new commit.